### DISCUSSION TITLE
Help for deployment ?

### DISCUSSION THREAD

**Discussion by chrisvidal** [OFFLINE-GmbH/oc-boxes-support#175] [Ideas and feature requests]
Is there a recommended way to deploy a local dev boxes on a staging or production server ?

Would a import/export of the boxes page be useful for that case ?

**Comment by tobias-kuendig** [comment]
The easiest way is to export all `offline_boxes_*` tables and any system files that belong to a `Box` model.

There was a import/export feature before, but it became unmaintainable once the revisions system and multisite support were added.
